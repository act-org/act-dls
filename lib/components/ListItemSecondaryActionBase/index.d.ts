/**
 * @prettier
 */
import * as React from 'react';
import { ListItemSecondaryActionProps } from '@material-ui/core';
export declare type Props = ListItemSecondaryActionProps;
declare const ListItemSecondaryActionBase: React.FC<Props>;
export default ListItemSecondaryActionBase;
//# sourceMappingURL=index.d.ts.map