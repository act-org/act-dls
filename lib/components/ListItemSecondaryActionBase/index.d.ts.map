{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["../../../src/components/ListItemSecondaryActionBase/index.tsx"],"names":[],"mappings":"AAAA;;GAEG;AAEH,OAAO,KAAK,KAAK,MAAM,OAAO,CAAC;AAC/B,OAAO,EAEL,4BAA4B,EAC7B,MAAM,mBAAmB,CAAC;AAE3B,oBAAY,KAAK,GAAG,4BAA4B,CAAC;AAEjD,QAAA,MAAM,2BAA2B,EAAE,KAAK,CAAC,EAAE,CAAC,KAAK,CAEmB,CAAC;AAErE,eAAe,2BAA2B,CAAC"}