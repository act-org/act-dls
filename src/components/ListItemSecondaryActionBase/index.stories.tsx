/**
 * @prettier
 */

import * as React from 'react';

import ListItemSecondaryActionBase from '.';

export const Base = (): React.ReactElement<any> => (
  <ListItemSecondaryActionBase>
    ListItemSecondaryAction
  </ListItemSecondaryActionBase>
);

export default {
  component: ListItemSecondaryActionBase,
  parameters: {
    info: {
      text:
        'This is the base variant of the ListItemSecondaryAction component.',
    },
  },
  title: 'Atoms/ListItemSecondaryAction',
};
