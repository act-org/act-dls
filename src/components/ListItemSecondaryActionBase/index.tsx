/**
 * @prettier
 */

import * as React from 'react';
import {
  ListItemSecondaryAction,
  ListItemSecondaryActionProps,
} from '@material-ui/core';

export type Props = ListItemSecondaryActionProps;

const ListItemSecondaryActionBase: React.FC<Props> = (
  props: Props,
): React.ReactElement<any> => <ListItemSecondaryAction {...props} />;

export default ListItemSecondaryActionBase;
