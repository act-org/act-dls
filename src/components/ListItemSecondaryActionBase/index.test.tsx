/**
 * @prettier
 */

import * as React from 'react';

import render from '~/helpers/test/render';

import ListItemSecondaryActionBase from '.';

describe('ListItemSecondaryActionBase', () => {
  it('matches the snapshot', () => {
    const { container } = render(
      <ListItemSecondaryActionBase>
        ListItemSecondaryAction
      </ListItemSecondaryActionBase>,
    );

    expect(container).toMatchSnapshot();
  });
});
