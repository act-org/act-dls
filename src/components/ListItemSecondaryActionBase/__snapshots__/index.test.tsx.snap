// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItemSecondaryActionBase matches the snapshot 1`] = `
<div>
  <div
    class="MuiListItemSecondaryAction-root"
  >
    ListItemSecondaryAction
  </div>
</div>
`;
